app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -69,6 +80,19 @@ fun ConversationRouterHomeBridge(
         onExpandBottomSheet = { onExpandHomeBottomSheet() },
         onScrollPositionChanged = onScrollPositionChanged
     )
+
+    MutingOptionsSheetContent(
+        mutingConversationState = mutingConversationState,
+        onItemClick = { conversationId, mutedStatus ->
+            viewModel.muteConversation(conversationId, mutedStatus)
+            conversationState.modalBottomSheetContentState.value.updateCurrentEditingMutedStatus(mutedStatus)
+        },
+        onBackClick = {
+            onExpandHomeBottomSheet()
+            mutingConversationState.closeMutedStatusSheetContent()
+            conversationState.modalBottomSheetContentState.value.updateCurrentEditingMutedStatus(mutingConversationState.mutedStatus)
+        }
+    )
```

Comment:
I do not think this should live inside the ConversationRouterHomeBridge() I am not 100 % sure but I think this will cause the Home TopBar to be not grayed out, so that the user can access the hamburger menu options. it is also abit confusing to me why it is being invoked on the end of the ConversationRouterHomeBridge I think a better place for it would be inside onHomeBottomSheetContentChange{ ... } we could have some flag living inside ConversationState that based on that flag we would invoke onHomeBottomSheetContentChange() with a different content for the bottom sheet. I think it it abit more readable because you would have some when() statement based on that flag and you see directly that based on the state we just rerender the bottom sheet content instead of having MutingOptionsSheetContent() floating kind of freely

Reply:
Makes sense, I tried to have this new sheet the most isolated possible, because we will have to reuse it (without the conversations menu options) later on, specifically in Other user profile and conversation details. 

I managed to get it somewhere in the middle, so now should be better =)